### PR TITLE
Fix png not returning from getCoverFromLink

### PR DESCRIPTION
**Pull request by antiss11**
https://github.com/antiss11/flibusta-api/issues/1

### CODE REVIEW

**Review by ynhhoJ — CHANGES_REQUESTED** [review]

File: src/flibustaOpdsApiHelper.ts [hunk 1]
```diff
@@ -49,7 +49,8 @@ class FlibustaOpdsApiHelper {
       return undefined;
     }
 
-    const result = links.find((link) => link['@_type'] === 'image/jpeg');
+    const result = links.find((link) => link['@_type'] === 'image/jpeg' ||
+                                        link['@_type'] === 'image/png');
```
[the remark this answers is not in the document]
Comment (antiss11) [reply src/flibustaOpdsApiHelper.ts:57]: Done accf2999483d4926e0e2b52aede7e9b9fe288f64.

File: src/flibustaOpdsApiHelper.ts [hunk 2]
```diff
@@ -123,14 +127,16 @@ class FlibustaOpdsApiHelper {
...
+        // const parsingOptions = {
+        //   ignoreAttributes: false,
+        //   unpairedTags: ['hr', 'br', 'link', 'meta'],
+        //   stopNodes: ['*.pre', '*.script'],
+        //   processEntities: true,
+        //   htmlEntities: true,
+        // };
+        const parser = new XMLParser({});
+        console.log(response.data);
+        // return new Promise(res => res(1));
```
Comment (ynhhoJ) [inline src/flibustaOpdsApiHelper.ts:139]: Why do you leave this code here?

File: src/flibustaOpdsApiHelper.ts [hunk 3]
```diff
@@ -49,7 +49,11 @@ class FlibustaOpdsApiHelper {
       return undefined;
     }
 
-    const result = links.find((link) => link['@_type'] === 'image/jpeg');
+    const result = links.find(
+      (link) => link['@_type'] === 'image/jpeg'
+      || link['@_type'] === 'image/png',
+    );
+
```
Comment (ynhhoJ) [inline src/flibustaOpdsApiHelper.ts:56]: Line 56 will cause [no-multiple-empty-lines](https://eslint.org/docs/rules/no-multiple-empty-lines) error, please, remove!

**Comment by ynhhoJ** [comment]
Please check your code before pushing it to main branch with ESLint

изображение

**Review by ynhhoJ — CHANGES_REQUESTED** [review]

**Comment by antiss11** [comment]
2022-09-07-183549_881x118_scrot
Fixed 14ea55fbfec686faeea16c93b58d3b03aaa354a8. Thanks for your response, it's a great experience for me!

**Review by ynhhoJ — APPROVED** [review]

**Comment by ynhhoJ** [comment]
@antiss11 , thank you fore your contribution. Merged!

**Pull request merged**